src/common/PreviewConfigFile.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { CommandLineUtils } from './Common';
+
+export class PreviewConfigFile {
+    public apps!: {
+        ios?: IOSAppPreviewConfig[];
+        android?: AndroidAppPreviewConfig[];
+    };
+
+    public getAppConfig(
+        platform: string,
+        targetApp: string
+    ): IOSAppPreviewConfig | AndroidAppPreviewConfig | undefined {
+        const appConfigs = CommandLineUtils.platformFlagIsIOS(platform)
+            ? this.apps.ios || []
+            : this.apps.android || [];
+
+        let config;
+
+        appConfigs.forEach((appConfig) => {
```

Comment:
It would actually be more appropriate in this case to use [`Array.prototype.find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) against `appConfigs`, rather than looping the contents.

Reply:
🤦 can't believe I forgot about `.find()`